packages/node_modules/@node-red/registry/lib/library.js

```diff
@@ -51,7 +51,9 @@ function getFlowsFromPath(path) {
                     }
                     i++;
                 })
-
+                if (!result.hasOwnProperty("f")) {
+                    reject(result);
+                }
```

Comment:
This change doesn't handle the case where a node has its examples in subdirectories under the examples directory:

```
├── examples
│   └── abc
│       └── flow.json
```

It sees that `examples` contains no files so rejects everything.

A more straightforward fix that would handle the top level `examples` directory being empty would be to check that neither `result.f` nor  `result.d`. But that wouldn't handle every case - such as having `examples/abc/` exist but no flow files.

A more complete fix would need to find all the files and then only work backwards to only include the dirs that in their path.

Reply:
OK - have done a more basic check for f: at any depth of the result... but doesn't remove any empty side branches but that is an edge edge case.